Mathlib/Algebra/DirectSum/Internal.lean

```diff
@@ -168,6 +169,15 @@ theorem coe_mul_apply_eq_dfinsuppSum [AddMonoid ι] [SetLike.GradedMonoid A]
 @[deprecated (since := "2025-04-06")]
 alias coe_mul_apply_eq_dfinsupp_sum := coe_mul_apply_eq_dfinsuppSum
 
+open Finset in
+theorem coe_mul_apply_eq_sum_antidiagonal [AddMonoid ι] [HasAntidiagonal ι]
+    [SetLike.GradedMonoid A] (r r' : ⨁ i, A i) (n : ι) :
+    (r * r') n = ∑ ij ∈ antidiagonal n, (r ij.1 : R) * r' ij.2 := by
```

Comment:
Does this follow slightly more quickly from `coe_mul_apply`, which looks to already be basically the `Antidiagonal` spelling?

Reply:
probably, will fix.